mentoring/mentor-guide.md

```diff
@@ -0,0 +1,100 @@
+  * Provide official feedback during intervals to Outreachy and the intern.
+  * Have read and understand the [rules and guidelines](https://www.outreachy.org/mentor/) around mentoring for Outreachy
+
+CNCF will financially support one Outreachy intern for Kubernetes; if you would like to participate and we already have one, assistance with securing a corporate sponsor for the budget (6.5k) may be needed.
+
+*[Google Summer of Code](google-summer-of-code.md)*
+* Being a mentor can take anywhere from 2-10 hours a week of your time depending on the student. In most cases, you will have a secondary/backup mentor to distribute the workload.
+* Before the Program:
+  * Read the official GSOC mentors manual to get an overview.
+  * Talk to interested candidates via Slack. Share with them the project details, goals and some ideas for implementation.
+  * Point them to self-contained, easy and newcomer-friendly bugs to fix.
+  * Review draft proposals and provide feedback before the application period.
+  * Evaluate the final proposals and select the student you want to mentor.
+* During the program:
+  * Set development and communication plans with your mentee. Make sure you establish realistic work objectives and timeline expectations.
+  * Help them get integrated into the community and encourage them to participate.
+  * Provide guidance such as pointers to useful documentation, code reviews, etc.
+  * Set aside time for a weekly sync via Slack or video call, whichever you prefer.
+  * Collect weekly status reports. They are important in making sure that time-line slippages and scope creep are addressed in a proactive manner.
+  * Keep track of the mentee's progress and keep them informed about their status. Make sure you deliver feedback early.
+  * Be responsive to questions - usual expectation is to reply within 24 hours (occasionally under 36 hours is ok).
+  * There will be three evaluations of your mentee's work. Submit them timely. The evaluation forms take hardly 10-15 min to fill.
+* After the program:
+  * Share your conclusions of the project in communication channels, wherever appropriate.
+  * Communicate to organization administrators any lessons learned.
+  * Recommend next steps to your mentee in the current project or wherever you think they might fit in our community.
+
+A former GSoC intern for Kubernetes is now leading this for us - mentoring works! Thanks @nikhita!
+
+*[Meet Our Contributors - Mentors On Demand](meet-our-contributors.md)*
+* One hour a month on a streaming YouTube show answering questions about the contribution process, your experience in open source, and other questions that people would normally ask to a mentor.
+* Typically a one hour a quarter time commitment is expected; this is an opt-in sign up.
+* You will need to be on the call 5 mins early to test audio quality and make sure we are good to go for streaming.
+
+### Benefits of being a mentor
```

Comment:
These trailing three sections now feel like a bit of an afterthought, where perhaps if they were at the start of the doc they encourage folks to keep reading and engage?